src/pages/Signup.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+// function sConsole(event) {
+//   event.preventDefault();
+//   const data = document.getElementById('LastName');
+//   console.log(data.value);
+// }
+const formData = new FormData(document.forms[0]);
+
+const obj = Object.fromEntries(Array.from(formData.keys()).map((key) => [key,
+  formData.getAll(key).length > 1 ? formData.getAll(key) : formData.get(key)]));
+
+console.log(`<pre>${JSON.stringify(obj)}</pre>`);
+function Signup() {
+  return (
+    <div>
+      <form>
+        <label htmlFor="FirstName">
+          First name:
+          <br />
+          <input type="text" id="signinform" name="FirstName" />
```

Comment:
instead of having id="signinform" for each input, it might be easier to have the id go into the label component- reduce repeated code!